### PR TITLE
SPIN-2071 Add RDS support

### PR DESCRIPTION
**Pull request by FishNecks**
- Allow minion to stop/start RDS databases
- Add clarifications to README

### CODE REVIEW

**Comment by tenyo** [comment]
I think you need to add the `databaseRunner` in `config.deco.json`.
Also, did you figure out why this caused Aurora MySQL database to fail over?
Otherwise looks good!

**Comment by FishNecks** [comment]
Thanks.

Added databaseRunner to .deco file.

The "failover" thing seems to be expected behavior when you start it back up. It's happening when you do the same thing from the Spinup UI, too.

File: docker/config.deco.json [hunk 1]
```diff
@@ -21,6 +21,14 @@
         "token": "{{ .instancerunner_token }}"
       }
     },
+    "databaseRunner": {
+      "type": "database",
+      "config": {
+        "endpointTemplate": "{{ .databaserunner_endpoint_tmpl }}",
+        "token": "{{ .databaserunner_token }}",
+        "auth_header": "X-Auth-Token"
```
Comment (tenyo) [inline docker/config.deco.json:29]: I don't think you can override this, it's the default

Comment (FishNecks) [reply docker/config.deco.json]: Yeah, forgot I made it the default in the databaserunner, I was looking at the servicerunner where it's set in the config. Fixed.

**Comment by tenyo** [comment]
Looks good 👍

File: jobs/databaserunner.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+			return "", NewRunnerError(ErrPreExecFailure, "template execution failed", err)
+		}
+
+		endpoint = out.String()
+		log.Debugf("parsed endpoint template: %s", endpoint)
+	}
+
+	switch action {
+	case "stop", "start":
+		j, err := json.Marshal(map[string]string{"State": action})
```
Comment (fishnix) [inline jobs/databaserunner.go:108]: this is really minor, but does the key need to be capitalized?  I'd rather see all of those types of things lowercased.

**Review by fishnix — APPROVED** [review]
Just that minor style question - otherwise looks good! 👍

File: jobs/databaserunner.go (hunk 2 above)
Comment (FishNecks) [reply jobs/databaserunner.go:108]: I was going off this: https://github.com/YaleSpinup/rds-api/blob/e6ca0b89aa5abd2d6f4f8a5eeb3a43e218d6e38b/README.md#stopping-and-starting-a-databasecluster

Comment (fishnix) [reply jobs/databaserunner.go:108]: Just try hitting it with postman?  If it supports lowercase, let's update the readme in the rds-api too :)

Comment (fishnix) [reply jobs/databaserunner.go:108]: The buffalo doc says it uses the standard library json encoding/decoding, so I think it should work:

https://gobuffalo.io/en/docs/bind#json-and-xml-binding

**Pull request merged**